src/main/java/seedu/duke/food/Food.java

```diff
@@ -0,0 +1,35 @@
+package seedu.duke.food;
+
+public class Food {
```

Comment:
Oops forgot to mention previously, remember to add JavaDoc for classes!

Reply:
Okay! :D